APIs_Platform/AT_CmdParser/README.md

```diff
@@ -0,0 +1,36 @@
+## ATCmdParser Example ##
+
+Example application showing usage of the `ATCmdParser` class with the ESP8266 WiFi module.
```

Comment:
I did not think we have README files with code snippets.

Reply:
@evedon the README is optional, although, most of the snippets include one. I've removed extraneous information and placed some of it in the docs section describing the snippet.